src/repositories/AdminRepository.ts

```diff
@@ -1,22 +1,25 @@
-import {getCollection} from "./connectToDB";
-import {Admin} from "./schemas";
+import { getCollection } from "./connectToDB";
+import { Admin } from "./schemas";
 
 export default {
-    async addAdmin(username: string, passwordHash: string) {
-        const result = await getCollection("admin").insertOne({username, passwordHash});
-        if (!result.acknowledged) {
-            throw Error("Admin could not be created")
-        }
-    },
-
-    async deleteAdmin(username: string) {
-        const result = await getCollection("admin").deleteOne({username})
-        if (result.deletedCount !== 1) {
-            throw Error("Admin could not be deleted")
-        }
-    },
+  async addAdmin(username: string, passwordHash: string) {
+    const result = await getCollection("admin").insertOne({
+      username,
+      passwordHash,
+    });
+    if (!result.acknowledged) {
+      throw Error("Admin could not be created");
+    }
```

Comment:
As we both seems to prefer single if-statements without brackets, you may as well remove them here.